sdks/java/io/common/pom.xml

```diff
@@ -33,10 +33,12 @@
       <dependency>
         <groupId>org.apache.beam</groupId>
         <artifactId>beam-sdks-java-core</artifactId>
+        <scope>test</scope>
```

Comment:
acknowledged. If we think HashingFn should move into main and/or core, I'd support that (probably not in this PR)